src/js/views/createCertificate/translations/en.createCertificate.i18n.json

```diff
@@ -14,7 +14,8 @@
     "generateCsrHelp": "How to Generate a CSR?",
     "csrHelpSteps": {
       "step1": "Step 1: ",
```

Comment:
The text has a white space, but for some reason it is not shown on screen.

![image](https://user-images.githubusercontent.com/36794274/153885342-f7735613-4ea4-46c3-93a3-b2eb3a0e70ba.png)